ibis/backends/tests/test_map.py

```diff
@@ -240,7 +240,11 @@ def test_literal_map_getitem_broadcast(backend, alltypes, df):
                 ),
                 pytest.mark.notyet(["pandas", "dask"]),
                 mark_notyet_postgres,
-                pytest.mark.notimpl("flink"),
+                pytest.mark.broken(
```

Comment:
```suggestion
                pytest.mark.notyet(
```

Reply:
JW: is there a heuristic for when to use the different pytest markers? I've been confused about when to use `pytest.mark.notyet` vs `pytest.mark.broken` vs `pytest.mark.notimpl`

Reply:
The ideas are scattered around a bit, should probably stick them into the dev docs.

* `notimpl`: the backend can do a thing, we haven't mapped the op
* `notyet`: the backend cannot do a thing, but might in the future
* `never`: the backend will never support this / pass this test (common example here is a test running on sqlite that relies on strong typing)
* `broken`: this test broke and it's demonstrably unrelated to the PR I'm working on and fixing it shouldn't block this PR from going in (but we should fix it up pronto)

Reply:
Thanks! I can put these into the dev docs